adminSiteClient/VariableEditPage.tsx

```diff
@@ -528,6 +516,54 @@ class VariableEditor extends React.Component<{
                         </form>
                     </section>
                 )}
+                <section>
+                    <h3>Partial Grapher Config</h3>
+                    <FieldsRow>
+                        <TextAreaField
+                            label="Grapher Config (edited via the ETL)"
+                            value={YAML.stringify(variable.grapherConfigETL)}
+                            disabled
+                            rows={8}
+                        />
+                        <div>
+                            <TextAreaField
+                                label="Grapher Config (edited in the admin)"
+                                value={YAML.stringify(
+                                    variable.grapherConfigAdmin
+                                )}
+                                disabled
+                                rows={8}
+                            />
+                            <a
+                                className="btn btn-primary"
+                                href={`/admin/variables/${variable.id}/config`}
+                                target="_blank"
+                                rel="noopener"
+                            >
+                                {variable.grapherConfigAdmin
+                                    ? "Edit"
+                                    : "Create"}
+                            </a>
+                            {variable.grapherConfigAdmin && (
+                                <button
+                                    type="button"
+                                    className="btn btn-danger ml-2"
+                                    onClick={async () => {
+                                        await this.context.admin.rawRequest(
+                                            `/api/variables/${variable.id}/grapherConfigAdmin`,
+                                            undefined,
+                                            "DELETE"
+                                        )
+                                        window.location.reload()
```

Comment:
While talking about this in our dev meeting, I suddenly realized that I had left this criminal line in here! I'll replace it with an optimistic update in the next few days... :)

Reply:
this seems to still be here :)